util/cron/publish-docker-images.bash

```diff
@@ -19,6 +19,7 @@ export CHPL_HOME=$(cd $CWD/../.. ; pwd)
 log_info "Setting CHPL_HOME to: ${CHPL_HOME}"
 
 start_docker
+docker login -u $3 -p $4
```

Comment:
It would probably be helpful to have a check that this succeeded and output an error if not, since it could fail due to Dockerhub problems.

Reply:
Yes, let me add a check.